src/components/ChangeThumbnailButton.tsx

```diff
@@ -121,6 +130,12 @@ const ChangeThumbnailButton: FC<ChangeThumbnailButtonProps> = ({
     }
   }, [newThumbnailContent, pollNewThumbnail, newThumbnailIsReady, values]);
 
+  useEffect(() => {
+    return () => {
+      mountedRef.current = false;
+    };
+  }, []);
+
```

Comment:
Fikser memory-leak error fra react

Reply:
tar gjerne et miniforedrag på denne :-)  Er det uppy som er problemet ?

Reply:
![image](https://user-images.githubusercontent.com/44363838/108362142-ac801980-71f3-11eb-9fb5-b97f3f72dca9.png)

Reply:
Hvis man er lyn kjapp og hopper ut av komponenter før async har gjort seg ferdig, så får man denne feilen.
Googlet feilen og kom til denne posten:
https://stackoverflow.com/questions/56450975/to-fix-cancel-all-subscriptions-and-asynchronous-tasks-in-a-useeffect-cleanup-f

Etter implementasjonen forsvant buggen.